CONTRIBUTING.md

```diff
@@ -25,22 +25,21 @@ We welcome [issues](https://github.com/Microsoft/VoTT/issues) and [pull requests
 ## Pull Requests
 
 1. Find an issue to work on, or create a new one
```

Comment:
Since we'll be converting over to master soon - we may want to consider just updating this to reference `master` branch instead of `v2`